upgrade_groupd.sh

```diff
@@ -26,156 +26,13 @@ function ui {
-  
-  echo "waiting for pods running....."
-  sleep 30
-
-  cat << EOF > asb-cr.yaml
-apiVersion: osb.openshift.io/v1alpha1
-kind: AutomationBroker
-metadata:
-  name: ansible-service-broker
-  namespace: ansible-service-broker
-EOF
-  oc create -f asb-cr.yaml -n ansible-service-broker
-
-  echo "waiting for pods running....."
-  sleep 30
-  if [ !$(oc get po -n ansible-service-broker | tail -n +2 |grep  -cvE "(Running|Completed)") ] ; then
-    echo -e  "\033[31m TSB install successfully, need a few minutes to wait asb ready !  \033[0m"
-  else
-    echo -e "\033[31m TSB installation failed , please manually install ASB on console！ \033[0m"
-  fi
-
-}
-function enableTSB {
-  echo "==================Install TSB...."
-  oc patch ServiceCatalogAPIServer cluster -p '{"spec":{"managementState": "Managed"}}' --type=merge
-  oc patch ServiceCatalogControllerManager cluster -p '{"spec":{"managementState": "Managed"}}' --type=merge
-  oc new-project template-service-broker
-  cat << EOF > tsbog.yaml
-apiVersion: operators.coreos.com/v1
-kind: OperatorGroup
-metadata:
-  name: template-service-broker-og
-  namespace: template-service-broker
-spec:
-  targetNamespaces:
-    - template-service-broker
-EOF
-
-  oc create -f tsbog.yaml -n template-service-broker
-
-  cat << EOF > tsbcsc.yaml
-apiVersion: operators.coreos.com/v1
-kind: CatalogSourceConfig
-metadata:
-  name: installed-community-template-service-broker
-  namespace: openshift-marketplace
-spec:
-  csDisplayName: Community Operators
-  csPublisher: Community
-  packages: templateservicebroker
-  targetNamespace: template-service-broker
-EOF
-  oc create -f tsbcsc.yaml -n openshift-marketplace
-
-  cat << EOF > tsbsubscription.yaml
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  labels:
-    csc-owner-name: installed-community-template-service-broker
-    csc-owner-namespace: openshift-marketplace
-  name: templateservicebroker
-  namespace: template-service-broker
-spec:
-  channel: alpha
-  installPlanApproval: Automatic
-  name: templateservicebroker
-  source: installed-community-template-service-broker
-  sourceNamespace: template-service-broker
-  startingCSV: templateservicebrokeroperator.v0.2.0
-EOF
-  oc create -f tsbsubscription.yaml -n template-service-broker
-
-  echo "waiting for pods running....."
-  sleep 30
-
-cat << EOF > tsb-cr.yaml
-apiVersion: osb.openshift.io/v1alpha1
-kind: TemplateServiceBroker
-metadata:
-  name: template-service-broker
-  namespace: template-service-broker
-EOF
-  oc create -f tsb-cr.yaml -n template-service-broker
-
-  echo "waiting for pods running....."
-  sleep 30
-  if [ !$(oc get po -n template-service-broker | tail -n +2 |grep  -cvE "(Running|Completed)") ] ; then
-    echo -e  "\033[31m TSB install successfully, need a few minutes to wait tsb ready !  \033[0m"
-  else 
-    echo -e "\033[31m TSB installation failed , please manually install TSB on web console!  \033[0m"
-  fi
-}
 
```

Comment:
Why we should delete these two functions?  Or you mean we won't test tsb or asb related case on console upgrade testing?

Reply:
Yes, don't plan to do this because we only want to make sure console is accessible after upgrade and ready for basic functions. Testing TSB/ASB features gives extra effort but without much value IMO.

Reply:
Agree. Thanks